content/articles/adding-domain.markdown

```diff
@@ -7,18 +7,31 @@ categories:
 
 # Adding a Domain
 
-When adding a domain to your account, you can choose to register, transfer, or just use our DNS service if the domain is already registered with another registrar.
+When adding a domain to your account, you can choose to register a new domain, transfer an existing domain you own from another registrar, or just use our DNS service if the domain is already registered with another registrar and you'd like to keep it there.
 
 <div class="section-steps" markdown="1">
 ##### To add a domain
 
 1.  Log in to DNSimple with your user credentials.
 1.  If you have more than one account, select the relevant one.
-1.  On the header, click the <label>Domains</label> tab.
-1.  Click <label>Add Domain</label>.
+1.  Click <label>Add Domain</label> from the Dashboard.
+  ![Adding a domain button](/files/add-a-domain.png)
 1.  Choose the correct flow for your needs.
+  ![Adding a domain](/files/adding-a-domain.png)
 </div>
 
-![Adding a domain](/files/adding-a-domain.png)
-
 You'll then be prompted for the name of the domain you want to add to your account.
+
+When registering a new domain, after adding the domain name, you'll be prompted to complete the following:
```

Comment:
I feel like these should be subsections, not just paragraphs - @kayla-broussard thoughts?

Reply:
@itsalyse I agree, less of a chance that it will get lost, or seem overwhelming

Reply:
Agreed 👍